cPP_DSC.adoc

```diff
@@ -4219,10 +4219,17 @@ The dependencies between SFRs implemented by the TOE are addressed as follows.
 |Rationale Statement
 
 |FCS_CKM.1 
-|[FCS_CKM_EXT.7 or FCS_COP.1] 
+|[FCS_CKM.2, FCS_CKM.5, FCS_CKM_EXT.7 or FCS_COP.1] 
```

Comment:
FCS_CKM_EXT.7 is not a dependency required by CC:2022, unless I'm misreading the errata. Where does it come from?

Reply:
I would have to check to be certain, but it was one listed in the crypto WG, so I left it.